utils/sendmail_tls.go

```diff
@@ -0,0 +1,86 @@
+package main

```

Comment:
for this, we could merge `sendmail` and `sendmail_tls` and use a command line. 

```
$ go run sendmail.go --tls
```

what do you think ?